pkg/actions/addon/addon.go

```diff
@@ -22,19 +20,11 @@ type Manager struct {
 	clusterProvider *eks.ClusterProvider
 	withOIDC        bool
 	oidcManager     *iamoidc.OpenIDConnectManager
-	stackManager    StackManager
+	stackManager    manager.StackManager
 	clientSet       kubeclient.Interface
 }
 
-//go:generate counterfeiter -o fakes/fake_stack_manager.go . StackManager
-type StackManager interface {
```

Comment:
These custom interfaces are actually kind of useful for seeing which parts of the `StackManager` are used around the codebase, could be a guide to splitting it up

Reply:
Yeah I don't disagree, me and @Callisto13 had a chat about wether to keep these or not in the issue https://github.com/weaveworks/eksctl/issues/2931#issuecomment-748109410, I'm quite torn.

Reply:
Mike does have a point that it could make things easier when we come to splitting that beast of a stack manager.

Reply:
It does, but like you pointed out in the issue it means loads of duplication in what is likely every package and if I had to guess would make people writing tests less likely 🤷  I'm leaning towards this at the moment as it enables everyone to very quick start adding tests